capella2polarion/converters/converter_config.py

```diff
@@ -39,20 +43,98 @@ class LinkConfig:
     link_field: str = ""
     reverse_field: str = ""
 
+    @staticmethod
+    def _force_config(
+        links: list[str | dict[str, t.Any]], role_prefix: str = ""
+    ) -> list[LinkConfig]:
```

Comment:
We should rename this function to something like generate_links_configs and make it public
```suggestion
    def generate_links_configs(
        links: list[str | dict[str, t.Any]], role_prefix: str = ""
    ) -> list[LinkConfig]:
    """Generate LinkConfigs based on a list of strings or a configuration dict."""
```